prow/config.yaml

```diff
@@ -5992,6 +6022,29 @@ periodics:
       hostPath:
         path: /mnt/disks/ssd0/docker-graph
 
+- name: ci-kube-deploy-build
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+      args:
+      - "--repo=k8s.io/$(REPO_NAME)"
```

Comment:
periodics don't have $(REPO_NAME), so specify your checkout path :-)

and also add `labels: preset-service-account: true` for log/aftifacts upload stuff